test/functionalTest/cases/722_unhardwire_format_cpr/query_forward_in_json.test

```diff
@@ -0,0 +1,249 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Query Forward In JSON
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Register E1/T1/A1 in CB, accumulator as CP, in XML and with notifyFormat set to JSON
+# 02. Register E1/T1/A2 in CB, accumulator as CP, in JSON
+# 03. Query E1/T1/A1 in CB, using XML - forwarded to accumulator in JSON
+# 04. Query E1/T1/A1 in CB - forwarded to accumulator in JSON
```

Comment:
Which format is used in "Query E1/T1/A1 in CB" ? I guess that JSON, but maybe is clearer tell so explicitly.